File: submissions/sm_015_lalit/week_03/day_3/session_2/count_vowels.js
input="lalit"
count=0
for (i=0;i<input.length;i++){
    if (input[i]=="a" ||input[i]=="e" ||input[i]=="i" ||input[i]=="o" ||input[i]=="u" ||input[i]=="A" ||input[i]=="E" ||input[i]=="I" ||input[i]=="O" ||input[i]=="U"){
        count=count+1
    }
}
console.log("Number of vowels =",count)

File: submissions/sm_114_sudhir/week_13/day_4/session_1/count_vowels.py
given_str = list(input().split(' '))

vowels = ["a","e","i","o","u"]
print(given_str)
count = 0
for i in range(len(given_str)):
    for it in range(len(given_str[i])):
        if given_str[i][it] == "a" or given_str[i][it] == "e" or given_str[i][it] == "i" or given_str[i][it] == "o" or given_str[i][it] == "u":
            count += 1

print(count)